### PR TITLE
WIP - mes-1885-catb-test-result-schema

### PR DESCRIPTION
**Pull request by ali-campbell**
Opening this up to ongoing review.  So far, candidate details and test report data have been added.

### CODE REVIEW

File: mes-test-schema/categories/schema-category-B.json [hunk 1]
```diff
@@ -61,5 +876,5 @@
     }
   },
   "additionalProperties": false,
-  "required": ["category", "id", "testData", "preTestDeclarations"]
+  "required": ["category", "id", "candidate", "testData", "preTest"]
```
Comment (johnmorleybjss) [inline mes-test-schema/categories/schema-category-B.json]: `preTest` is the wrong key here.

Comment (ali-campbell) [reply mes-test-schema/categories/schema-category-B.json]: Changed

**Comment by ammarhaiderbjss** [comment]
@ali-campbell Great start! 👍 Is it still WIP so we can add in the post test stuff, e.g. write-up

**Comment by ali-campbell** [comment]
> @ali-campbell Great start! 👍 Is it still WIP so we can add in the post test stuff? e.g. write-up

Yes, still got a few test characteristic items to add as well.  May look at restructuring the test elements in a more meaningful way too.

**Comment by ali-campbell** [comment]
> @ali-campbell Great start! 👍 Is it still WIP so we can add in the post test stuff? e.g. write-up

Yes, still got a few test characteristic items to add as well.  May look at restructuring the test elements in a more meaningful way too.  Just need to get something merged in as a starting point for @johnmorleybjss.

**Pull request merged**